# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mapped_hyph/README.md
# Commit: a1ab26abf3eb
# Full Hash: a1ab26abf3eb645e202d69dffea86398a0ea4563
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-09-30 21:45:29
# Regressor Bug: 1667192
# File Overlap Count: 6
# Description:
#   Bug 1667192 - patch 1.1 - Run `mach vendor rust` to update in-tree mapped_hyph source. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91409
# ==============================================================================

diff -r 66b86f8b6682 -r a1ab26abf3eb third_party/rust/mapped_hyph/README.md
--- a/third_party/rust/mapped_hyph/README.md	Wed Sep 30 12:51:17 2020 +0000
+++ b/third_party/rust/mapped_hyph/README.md	Wed Sep 30 12:51:54 2020 +0000
@@ -46,7 +46,7 @@
 
 ## C and C++ bindings
 
-See the `mapped_hyph.h` header for C/C++ APIs that can be used to load hyphenation files
+See src/ffi.rs for C/C++ APIs that can be used to load hyphenation files
 and to locate valid hyphenation positions in a word.
 
 ## Sample programs
@@ -63,6 +63,21 @@
 
 ## Release Notes
 
+### 0.4.0
+
+* Added a boolean `compress` param to the pattern compiler to control whether
+  it attempts to compress the compiled table by merging duplicate states (which
+  takes significant extra time).
+
+* Added FFI functions to compile hyphenation tables from a file path or a buffer,
+  intended for use from Gecko.
+
+### 0.3.0
+
+* Switched from MPL2 to Apache2/MIT dual license.
+
+* Misc bug-fixes and optimizations.
+
 ### 0.2.0
 
 * Implemented a hyphenation table compiler in the `builder` submodule,